Framework/PythonInterface/mantid/kernel/src/Exports/VMD.cpp

```diff
@@ -104,9 +104,12 @@ void export_VMD() {
       .def(self + self)
       .def(self += self)
       .def(self - self)
+      // cppcheck-suppress duplicateExpression
```

Comment:
These are better addressed by following the example @AndreiSavici gave in issue #15281.